.gitpod.Dockerfile

```diff
@@ -1,4 +1,26 @@
-FROM gitpod/workspace-full
+FROM gitpod/workspace-base
+
+# Installing Node
+# https://github.com/nvm-sh/nvm#installing-and-updating
+USER gitpod
+ENV NODE_VERSION=16.13.0
+RUN curl -fsSL https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash \
+    && bash -c ". .nvm/nvm.sh \
+        && nvm install $NODE_VERSION \
+        && nvm alias default $NODE_VERSION \
+        && npm install -g typescript yarn pnpm"
```

Comment:
Why `yarn`?

Reply:
Also, why install TS globally?  I think we install it as part of the web front-end, so this shouldn't be needed.  Only `pnpm` is needed globally.

Reply:
I was just going off of `yarn dev` here: [src/web/readme.md](https://github.com/Seneca-CDOT/telescope/blob/c092a7f2571d859a50a077b6f7d00f11f44524ba/src/web/README.md?plain=1#L10)

Is it not needed?